base/tools/src/main/java/com/netscape/cmstools/nss/NSSCertImportCLI.java

```diff
@@ -84,11 +84,12 @@ public void execute(CommandLine cmd) throws Exception {
             throw new Exception("Unsupported format: " + format);
         }
```

Comment:
Slightly less of a problem here, but for consistency would be nice to do something similar.

Reply:
Done.